WooCommerce/Classes/ViewRelated/ReusableViews/UnitInputTableViewCell.swift

```diff
@@ -30,6 +30,10 @@ final class UnitInputTableViewCell: UITableViewCell {
     private var inputFormatter: UnitInputFormatter?
     private var onInputChange: ((_ input: String?) -> Void)?
 
+    /// The value inside `inputTextField` exposed outside of the cell
+    ///
+    private (set) var value: String = ""
```

Comment:
would it work if this just returns the text field's text like below so that we don't have to update it whenever it's set or changed?

```swift
var value: String {
  inputTextField.text ?? ""
}
```